### DISCUSSION TITLE
OBS Studio for M1

### DISCUSSION THREAD

**Discussion by JDftw** [obsproject/obs-studio#5575] [Development Questions] [answered]
Hi! Would like to ask if OBS Studio works on M1? Looking forward to it! Thank you! :)

**Comment by WizardCM** [comment, marked as answer]
There are a number of moving parts to this, but things are progressing.

The current plan (which may change, but it's looking good) is to release a "beta" M1 build alongside OBS 27.2 ([and CEF 95](https://github.com/obsproject/obs-browser/pull/323)) soon for wide-scale testing, and then properly release it for OBS 28.0 with the [CMake rewrite](https://github.com/obsproject/obs-studio/pull/4560).

In the meantime, Rosetta 2 works quite well and is a solid intermediary until then.

**Reply by JDftw** [reply]
Thank you! :)

**Comment by Developer-Ecosystem-Engineering** [comment]
Looks like the parts have all come together [here](https://github.com/obsproject/obs-studio/pull/5155)

**Comment by kvzonbr** [comment]
Looks good for this update, (i'm writing here se we don't open another issue), I was looking on the 27.2 Beta 2 files, and was wondering if will be a M1 Build in this release?

**Reply by dodgepong** [reply]
No, it will more likely come in version 28 later this year.

**Comment by bora89** [comment]
Guys, how are things going nowadays for M1 support?

**Reply by RytoEX** [reply]
OBS Studio builds from CI have been M1/ARM native since https://github.com/obsproject/obs-studio/pull/5155 was merged. The latest build can always be found here:
* https://github.com/obsproject/obs-studio/actions/workflows/main.yml?query=branch%3Amaster+is%3Acompleted

Select the latest build, scroll down, and look for the arm64 build for macOS.
